jenkins/Jenkinsfile.daily

```diff
@@ -0,0 +1,45 @@
+pipeline {
```

Comment:
I wasn't able to combine `main` daily builds with `rel-` daily builds into the same dispatcher because we'd need two `cron` timers: a daily one for `main`, and a monthly one for `rel-*`, and there isn't a way for the stages to determine which timer fired to know whether to fire off a `main` daily build or a `rel-*` monthly build.

So a daily dispatcher has been added back